<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IQSetupWSLDistributionWizard</class>
 <widget class="QDialog" name="IQSetupWSLDistributionWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>WSL distribution label:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leWSLLabel">
       <property name="toolTip">
        <string>The label of the distribution to be created. I can be freely chosen but it should be unique.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Linux user to create:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leWSLUser">
       <property name="toolTip">
        <string>This user name can be freely chosen though it must not collide with one of the linux system accounts.</string>
       </property>
       <property name="text">
        <string>user</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Repository URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRepoURL">
       <property name="toolTip">
        <string>The default should match. Please change this only, if you know what you are doing.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCredentials">
     <property name="title">
      <string>Repository credentials</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>User name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRepoUser">
          <property name="toolTip">
           <string>Please enter the credentials which you received.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRepoPwd">
          <property name="toolTip">
           <string>Please enter the credentials which you received.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Progress</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusText">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IQSetupWSLDistributionWizard</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IQSetupWSLDistributionWizard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
